plugin-commons/src/main/java/org/sonarsource/sonarlint/core/plugin/commons/PluginsLoader.java

```diff
@@ -79,10 +81,17 @@ public PluginsLoadResult load(Configuration configuration) {
     var instancesLoader = new PluginInstancesLoader();
     var pluginInstancesByKeys = instancesLoader.instantiatePluginClasses(nonSkippedPlugins);
 
-    return new PluginsLoadResult(new LoadedPlugins(pluginInstancesByKeys, instancesLoader, maybeDbdAllowedPlugins(configuration.enableDataflowBugDetection)),
+    return new PluginsLoadResult(new LoadedPlugins(pluginInstancesByKeys, instancesLoader, additionalAllowedPlugins(configuration)),
       pluginCheckResultByKeys);
   }
 
+  private static Set<String> additionalAllowedPlugins(Configuration configuration) {
+    var allowedPluginsIds = new HashSet<String>();
+    allowedPluginsIds.add("textenterprise");
```

Comment:
Will it work when we allow the `sonar-text-enterprise` but SQ does not have it in stock? Like community or older editions?

Reply:
Yes: this is only a list used to filter which plugins are allowed among the ones downloaded from the server.

Reply:
Perfect, I wasn't quite sure about this.